tests/test_analytical_solution.py

```diff
@@ -19,23 +21,38 @@
 # Model specifications
 # ======================================================================================
 
-ISKHAKOV_2017_PARAMS = {
-    "beta": 0.98,
-    "utility": {"disutility_of_work": None},
-    "next_wealth": {
-        "interest_rate": 0.0,
-        "wage": 20.0,
-    },
-}
+
+def _get_iskhakov_2017_model(n_periods):
+    model_config = deepcopy(BASE_MODEL_WITH_FILTERS)
+    model_config["n_periods"] = n_periods
+    # remove age and wage functions, as they are not modelled in Iskhakov et al. (2017)
+    model_config["functions"] = {
+        name: func
+        for name, func in model_config["functions"].items()
+        if name not in ("age", "wage")
+    }
+    return model_config
+
+
+def _get_iskhakov_2017_params(disutility_of_work):
```

Comment:
Why don't we make all parameters modifiable and leave the defaults at the values found in the main specification of the paper?